### PR TITLE
Updated the Markdown file.

### PR DESCRIPTION
**Pull request by rsal0007**
# Description

Updated the markdown (readme.md) file as per guidelines mentioned in Milestone 4.

# Type of change

- [ ] Bug fix
- [ ] New feature
- [ ] Refactoring
- [x] Documentation
- [ ] Improvement

# Risk

Low.

### CODE REVIEW

**Review by AkashSaggar — APPROVED** [review]

**Review by melvinalex7 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,6 +1,13 @@
 # Researchify
 
-An open source lightweight CMS enabling streamlined webpage development for academic research groups.
+_An open source lightweight CMS enabling streamlined webpage development for academic research groups_
+
+Researchify is a web platform for academic researchers, providing an end-to-end service to present and share their publications and other information on their own website. As such our product includes not only Researchify, the web platform, but also the development of websites deployed by users. 
+
+To provide this service, this repository is set up similarly to the components depicted in Figure 2.
```
Comment (aksh0001) [inline README.md]: nit: add Figure 2 or avoid saying it?

File: README.md [hunk 2]
```diff
@@ -1,6 +1,13 @@
...
+
+Researchify is a web platform for academic researchers, providing an end-to-end service to present and share their publications and other information on their own website. As such our product includes not only Researchify, the web platform, but also the development of websites deployed by users. 
+
+To provide this service, this repository is set up similarly to the components depicted in Figure 2. 
+- The ‘client’ folder contains everything related to the Researchify web platform. Notable items include the React UI components of Researchify, along with associated Redux reducers and actions all within their own separate subfolders.
+- The ‘api’ folder contains all backend related functionality, which are implemented as Express REST APIs using Node.JS.
+- The ‘scholly’ folder contains 2 subfolders ‘base’ and ‘service’. The ‘base’ folder houses the website layouts implemented using React components and environment variables for dynamic data rendering. The ‘service’ folder contains an independent microservice that invokes the base application to build a client website and deploys it to GitHub pages.
```
Comment (aksh0001) [inline README.md:10]: Do we want to add READMEs to the scholly base/service folders as well, containing the info + run instructions? It'd be a nice addition, but this is all g.
https://github.com/Researchify/Researchify/tree/main/scholly/base
https://github.com/Researchify/Researchify/tree/main/scholly/service

**Review by aksh0001 — APPROVED** [review]

**Review by cyr17 — APPROVED** [review]
EZ

File: README.md (hunk 1 above)
Comment (rsal0007) [reply README.md]: Yea good catch!

**Pull request merged**